extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -340,7 +340,8 @@ export function getCurrentCommandAndArgs(commandLine: string, cursorPosition: nu
 	let lastResetIndex = -1;
 	for (const char of resetChars) {
 		const idx = beforeCursor.lastIndexOf(char);
-		if (idx > lastResetIndex) {
+		// Ensure the prior index is a space
+		if (idx > 0 && (beforeCursor[idx - 1] === ' ')) {
```

Comment:
I think this could cause some things to not work properly. `&&` without a space works for example. How about we just change `"-in"` and the like to `" -in "` as I think it must be like that for pwsh.